.gitignore

```diff
@@ -1,30 +0,0 @@
-# Built application files
-*.apk
-*.ap_
-
-# Files for the Dalvik VM
-*.dex
-
-# Java class files
-*.class
-
-# Generated files
-bin/
-gen/
-
-# Gradle files
-.gradle/
-build/
-
-# Local configuration file (sdk path, etc)
-local.properties
-
-# Proguard folder generated by Eclipse
-proguard/
-
-# Log Files
-*.log
-
-# Intellij Files
-.idea/
-*.iml
```

Comment:
This .gitignore is substantially more useful than the one it was replaced with. Please restore this one.